packages/fuel-indexer-utils/src/utilities.rs

```diff
@@ -0,0 +1,37 @@
+use crate::prelude::sha256_digest;
+use crate::prelude::{Bytes32, SizedAsciiString};
+
+pub fn u64_id(d: &[u8; 8]) -> u64 {
```

Comment:
I realize we don't have much to put in `fuel-indexer-utils` yet, but could we rustdoc what we have? I think the SDK's rustdocs are a good example of what I'm referring to.

Ideally, users would only ever look up docs for this `fuel-indexer-utils` crate, so of all our crates, this one should be the easiest to follow (for example on crates.io)

Reply:
Oops, forgot about that bit; will do.